metabase/metabase
# Add static funnel bar and combined scalars charts

Closes https://github.com/metabase/metabase/issues/35823

### Description

Adds two static viz visualizations using ECharts:
- Funnel bar chart which is rendered when a Funnel chart has `funnel.type: "bar"`
- Combined scalars chart which is rendered when a dashcard combines >1 scalar

### How to verify

To test frontend run storybook and verify both FunnelBarChart and ScalarChart under the static viz group

### Demo

**Combined scalars**
<img width="555" alt="Screenshot 2024-01-12 at 12 08 33 AM" src="https://github.com/metabase/metabase/assets/14301985/37ae2f66-229a-4052-8e96-a2f0dce9da1e">

**Funnel bar**
<img width="553" alt="Screenshot 2024-01-12 at 12 08 14 AM" src="https://github.com/metabase/metabase/assets/14301985/4a507159-e739-4dd1-bd99-7bb20e3cfff1">

### Checklist
